lib/tasks/migrate_to_pg.rake

```diff
@@ -139,6 +139,16 @@ namespace :migrate_to_pg do
         attrs
       end
       migrate_submodel(pt, mongo_pt, pg, mongo, 'notes', 'patient', extra_transform)
+
+      # Finally, clean up events - creating new objects fires events callbacks;
+      # we want to just change IDs on the old ones
+      # Start by nuking old events
+      Event.where('length(patient_id) != 24').destroy_all # anything with a letter is a BSON ID
+      # Then change ids of what's left
+      puts "Updating patient ids on #{Event.count} events"
+      Event.all.each do |event|
+        event.update patient_id: Patient.find_by(mongo_id: event.patient_id).id
+      end
```

Comment:
does this make all the events look new? 🤔 

Before
```
irb(main):001:0> Event.first
=> #<Event id: 1, cm_name: [FILTERED], event_type: "reached_patient", line: "DC", patient_name: [FILTERED], patient_id: "60c
0184545cb72b055df1fb2", pledge_amount: nil, created_at: "2021-06-09 01:24:21", updated_at: "2021-06-09 01:24:21">
irb(main):002:0> Event.last
=> #<Event id: 158, cm_name: [FILTERED], event_type: "reached_patient", line: "DC", patient_name: [FILTERED], patient_id: "6
0c0184845cb72b055df21a0", pledge_amount: nil, created_at: "2021-06-09 01:24:24", updated_at: "2021-06-09 01:24:24">
```

After:
```
irb(main):001:0> Event.first
=> #<Event id: 1, cm_name: [FILTERED], event_type: "reached_patient", line: "DC", patient_name: [FILTERED], patient_id: "1",
 pledge_amount: nil, created_at: "2021-06-09 01:24:21", updated_at: "2021-06-09 01:28:09">
irb(main):002:0> Event.last
=> #<Event id: 158, cm_name: [FILTERED], event_type: "reached_patient", line: "DC", patient_name: [FILTERED], patient_id: "3
9", pledge_amount: nil, created_at: "2021-06-09 01:24:24", updated_at: "2021-06-09 01:28:09">
```

Looks like the `created_at` is retained and the `updated_at` is updated (natch). Does that matter?

  - ✅ Events Controller - cares about `created_at`  
  - ✅ Events Helper - cares about `created_at`
  - ✅ Events Model - cares about `created_at`
  - ✅ No other objects care about the Event's timestamps

all clear. 🎉 
and since these are ephemeral objects that delete after three weeks I'm not worried about them having distinctive updated timestamps